metastats-domain/src/main/groovy/life/qbic/metastats/filter/FilterExperimentDataFileOutput.groovy

```diff
@@ -0,0 +1,15 @@
+package life.qbic.metastats.filter
+
+import life.qbic.metastats.datamodel.MetaStatsPackageEntry
+
```

Comment:
Please add interface, author and version information

Reply:
Do you mean at the top of the interface?

Reply:
yes exactly, as JavaDoc comment

Reply:
done with 10ca084